tests/test_cases/test_cocotb/test_scheduler.py

```diff
@@ -263,3 +264,81 @@ async def waiter(event):
     await NullTrigger()
 
     assert waiter_scheduled is True
+
+
+@cocotb.test()
+async def test_task_repr(dut):
+    """
+    Test RunningTask.__repr__
+    """
```

Comment:
```suggestion
    """Test RunningTask.__repr__"""
```
(Our tests are not usually that polished, but why not start :)